src/Tribe/Privacy.php

```diff
@@ -54,4 +56,176 @@ public function default_privacy_policy_content() {
 		return apply_filters( 'tribe_tickets_default_privacy_policy_content', $content );
 
 	}
+
+	/**
+	 * Register exporter for Tickets attendees saved data.
+	 *
+	 * @since TBD
+	 * @param $exporters
+	 *
+	 * @return array
+	 */
+	public function register_exporter( $exporters ) {
+		$exporters[] = array(
+			'exporter_friendly_name' => __( 'Event Ticket RSVP Atendee' ),
```

Comment:
"Event Tickets" and "Attendee"

Reply:
Great catch, the singular for "ticket" was on purpose though. 😂